source/rmarkdown/analyses_diversity/analyses_diversity.Rmd

```diff
@@ -67,10 +68,40 @@ metadata <- readr::read_csv(
     diepte = gsub("_|/", "-", diepte) |> factor()
   )
 
-combined <- diversiteit %>%
-  inner_join(metadata, by = "sample")
 ```
 
+In `mbag_combined_dataframe.csv` zitten de samples zonder taxa niet in (na te vragen)!
+
+```{r echo=TRUE}
+sum(diversiteit$observed == 0)
+```
+
+Deze nulwaarnemingen moeten we terug toevoegen (observed wordt dan 0, maar Shannon en Simpson zijn dan niet gedefinieerd).
```

Comment:
@hansvancalster this is not the case for the `Nematoda - 18s - asv` data from ILVO, since that is an unusual case see https://github.com/inbo/mbag-bodem/pull/32#issuecomment-2520109557

and in general, I realize this can differ per `primerset` (primers that target more than one group vs group-specific), and `sample` (for some samples total read count across everything a primer captures is zero, in which case I think we should assume the eDNA pipeline in the lab failed)

Reply:
Thanks for documenting this in an issue. One quick question: could a total read count for a sample be zero because it is completely denuded of everything the primer targets? I'm thinking of pesticide misuse and other forms of pollutions. If that could be the case, we should be careful in attributing it to a failed eDNA pipeline. Are there ways to know this?

Reply:
Good question. I know that many people (e.g. ILVO and the LUCAS project) resequence a sample if it has less than 50K total reads across everything a primer captures, but I have often wondered whether total read count is really random in general? I will continue the discussion in https://github.com/slambrechts/INBO_eDNA_metabarcoding_BODEM/issues/238